### PR TITLE
[DC-3443] Add notebook check to ensure withdrawn dup participants do not have any data in the CDR

### PR DESCRIPTION
**Pull request by lledezma**

### CODE REVIEW

**Review by nishanthpp93 — COMMENTED** [review]
Looks good but want to clarify a couple of things.

File: data_steward/analytics/cdr_ops/cross_ct_rt/CDR_QC_RT_vs_CT.py [hunk 1]
```diff
@@ -634,10 +637,73 @@
...
+    SELECT
+        \{{table_name}}\ AS table_name,
+        COUNT(*) AS total_records
+    FROM
+        `{{project_id}}.{{dataset_id}}.{{table_name}}`
+    WHERE person_id IN (
+        SELECT 
+            pr.research_id
+        FROM
+           `{{project_id}}.{{pipeline_tables}}.{{pid_rid_mapping}}` AS pr
```
[the remark this answers is not in the document]
Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/cross_ct_rt/CDR_QC_RT_vs_CT.py:679]: The new table should come through the new RDR export. The one that was generated after October 1st(cut-off date)

File: data_steward/analytics/cdr_ops/cross_ct_rt/CDR_QC_RT_vs_CT.py [hunk 2]
```diff
@@ -634,10 +637,73 @@
...
+        JOIN
```
[the remark this answers is not in the document]
Comment (lledezma) [reply data_steward/analytics/cdr_ops/cross_ct_rt/CDR_QC_RT_vs_CT.py:680]: If there is no pid_rid mapping for a pid, then `pr.research_id ` will return a NULL value with an outer join. So, I think it is safe to assume that every pid will be de-identified and verified by this point.

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

File: data_steward/analytics/cdr_ops/cross_ct_rt/CDR_QC_RT_vs_CT.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nishanthpp93) [reply data_steward/analytics/cdr_ops/cross_ct_rt/CDR_QC_RT_vs_CT.py:679]: Also @ksdkalluri, we don't need to use rdr.pid_rid_mapping for this, correct?

**Pull request merged**